models/favorite.js

```diff
@@ -1,17 +1,8 @@
-// Dependencies
-// =============================================================
-var Sequelize = require("sequelize");
-var db = require("../config/connection.js");
-var User = require("../models/users.js");
+module.exports = function (sequelize, DataTypes) {
+    var Favorite = sequelize.define("Favorites", {
+        animalId: DataTypes.STRING
+    });
+    // Favorite.belongsTo(Users)
```

Comment:
We should probably keep this as it adds the `userId` to the table.
To test I created a user with this query:
```
INSERT into purrfect_match_DB.users (username, password, name, location, favorites, preferences, has_preferences) 
values ("toto", "user-id", "toto", "San Francisco", null, null, false);
```